lua/apisix/core/table.lua

```diff
@@ -81,4 +82,18 @@ end
 _M.deepcopy = deepcopy
 
 
+function _M.readonly(t)
```

Comment:
This function prohibits the insertion of new elements, but should it still be possible to modify existing data?

Reply:
> should it still be possible to modify existing data?

Modification of existing data is not allowed. I'll add new test case to confirm this.

Reply:
added more test case: https://github.com/apache/incubator-apisix/pull/1143/commits/9d0ffbe6cefe7dbd0ba70ae45baab007c89a39e5#diff-7e4b33723fb43f3a64bd46cb434958e7R67